.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,51 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  pull-requests: read
+  # Optional: Allow write access to checks to allow the action to annotate code in the PR.
+  checks: write
+
+jobs:
+  golangci:
+    strategy:
+      matrix:
+        go: ['1.21']
```

Comment:
Any benefit to linting against 1.22 as well?

Reply:
I choose 1.21 because your `go.mod` is fixed to 1.21

https://github.com/Boeing/config-file-validator/blob/c2b54549fba58a23b758902e8518e8ed1b306558/go.mod#L3

Reply:
but I now notice that gorelease package with 1.22

https://github.com/Boeing/config-file-validator/blob/c3cb101ecd5f02dc3ca6156fc4bdea5afc0d64c0/.github/workflows/release.yml#L36

but you are not using it fully because of 1.21 fixed in go mod

Reply:
It's actually just a declaration of the minimum version [ref](https://go.dev/doc/modules/gomod-ref#go) and I upgraded everything else to 1.22 about a month ago in [this PR](https://github.com/Boeing/config-file-validator/pull/116). I'd suggest either bumping it up to 1.22 or adding 1.22. I'm also fine just leaving it at 1.21 for now so this can get merged